index.html

```diff
@@ -174,14 +174,25 @@ <h3>LET'S CONNECT</h3>
         </section>
 
         <section class="section-3">
-            <h2>Contacts me</h2>
+        <h2>Contacts me</h2>
             <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
-            <div class="input">
-                <input class="input-1" type="text">
-                <input class="input-1" type="text">
-                <input type="text" name="" class="input-2" placeholder="Write your message here">
-                <button class="btn" type="button"><a href="#">Get it Touch</a></button>
-            </div>
+            <form action="#" method="post">
+                <fieldset class="contact-form">
+                    <label for="username">
+                        <input type="text" class="input-contact" required name="username" maxlength="30" placeholder="Username">
+                    </label>
+                    <label for="email">
+                        <input type="email" class="input-contact" required name="email" placeholder="name@domain.com">
+                    </label>
+                    <label for="message">
+                        <textarea name="message" class="msg" maxlength="500" placeholder="Write your message here"></textarea>
+                    </label>
+                    <div class="button">
+                      <button type="submit" class="btn last-btn">Get in touch</button>
+                    </div>
+                </fieldset>
+
+            </form>
```

Comment:
You did a good job working on this milestone! 💯 However, there's still some room to improve.

- [ ] 🚧 According to the requirements, you should use the `Formspree` service to collect data, please refer to their [documentation](https://formspree.io/html/) and implement this feature in your form! 🤓 🔧 

Overall, the design is good, but there are small issues that you could easily solve in order to make it even better. 🔧 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/212986022-6f6d5329-e3fa-49cc-b060-5309106bcbd5.png) | ![image](https://user-images.githubusercontent.com/77069011/212985850-aeb123b2-cfcc-467b-81f4-f8bed76494df.png) |

- [ ] 🚧 Please make sure to use the correct background image, as you can see, it is different from the one provided in the Figma template. 🔧 

- [ ] 🚧 Consider fixing the typo in the title `"Contacts"` ==> `"Contact"`. 🔧 

- [ ] 🚧 Please add some padding to your input fields, the text is very close to the edges! 🔧 

Applying those changes will improve the look and feel of this section of your page, so try to spend some time working on them! 🤓 🛠️